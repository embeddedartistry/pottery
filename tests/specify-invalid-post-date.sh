pottery init
pottery note -t 2018-08-2314:01:23Z The new note
find docs/project-history
