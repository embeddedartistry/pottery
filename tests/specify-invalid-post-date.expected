pottery init
pottery note -t 2018-08-2314:01:23Z The new note
Invalid date time
find docs/project-history
docs/project-history
